release-notes/README.md

```diff
@@ -23,12 +23,16 @@ and bad.  We would love to hear from you.
 
 Below are the release notes for the latest ODPi Egeria release:
 
- * [Release 2.1](release-notes-2-1.md) - July 2020
+ * [Release 2.2](release-notes-2-2.md) - August 2020
```

Comment:
I assume we want September

Reply:
It's debatabe. so far I've followed the pattern in previous releases - the date represents when we cut the branch & have developed the work. The release lags a little - ideally it would be end of month, but this release things went on for longer due to some issues with the containers & janusgraph. So I would stick with August.